core/tracepoint/enabled_spec.rb

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint#enabled?' do
+    it 'returns the current status of the trace' do
```

Comment:
We need a test that shows that `enabled?` is `true` when enabled too... Otherwise `def enabled?; false; end` is an implementation that would pass the specs!

Reply:
yes, we do need the test. I have updated the PR.